aeron-driver/src/main/c/aeron_driver_context.c

```diff
@@ -1420,6 +1420,45 @@ int aeron_driver_context_validate_mtu_length(uint64_t mtu_length)
     return 0;
 }
 
+int aeron_driver_context_validate_receiver_window_length(const char* param, size_t window_length, size_t mtu_length)
+{
+    if (window_length < (mtu_length << 1))
+    {
+        AERON_SET_ERR(
+            EINVAL,
+            "%s=%" PRIu64 " must be at least two times larger than the %s=%" PRIu64,
+            param,
```

Comment:
You should cast the `size_t` to a `uint64_t` if you are using the `PRIu64` format options.  The other option is `%z` which is explicitly for `size_t`, but we have generally avoided that.